prow/cmd/deck/main.go

```diff
@@ -603,8 +611,10 @@ func handleProwJobs(ja *jobs.JobAgent) http.HandlerFunc {
 		// If we have a "var" query, then write out "var value = {...};".
 		// Otherwise, just write out the JSON.
 		if v := r.URL.Query().Get("var"); v != "" {
```

Comment:
Could we DRY this bit out as well while we're here? A helper that checks the query and writes the appropriate thing

Reply:
Sounds good to me. I'll improve the patch.